### PR TITLE
spec: typos & other little fixes

### PR DESCRIPTION
**Pull request by zramsay**
typo and grammatical fixes, etc

### CODE REVIEW

File: docs/specification/new-spec/p2p/connection.md [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
 
 ### Ping and Pong
 
 The ping and pong messages consist of writing a single byte to the connection; 0x1 and 0x2, respectively.
 
-When we haven't received any messages on an `MConnection` in a time `pingTimeout`, we send a ping message.
+When we haven't received any messages on an `MConnection` in time `pingTimeout`, we send a ping message.
 When a ping is received on the `MConnection`, a pong is sent in response only if there are no other messages
-to send and the peer has not sent us too many pings.
+to send and the peer has not sent us too many pings (how many is too many?).
```
Comment (melekes) [inline docs/specification/new-spec/p2p/connection.md]: > how many is too many?

this is not implemented yet

File: docs/specification/new-spec/p2p/node.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-A Tendermint P2P network has different kinds of nodes with different requirements for connectivity to others.
+A Tendermint P2P network has different kinds of nodes with different requirements for connectivity compared to other types of networks.
 This document describes what kind of nodes Tendermint should enable and how they should work.
 
 ## Seeds
 
 Seeds are the first point of contact for a new node.
-They return a list of known active peers and disconnect.
+They return a list of known active peers and disconnect....if?
```
Comment (melekes) [inline docs/specification/new-spec/p2p/node.md]: > ....if?

no if. they disconnect always

File: docs/specification/new-spec/spec-notes.md [hunk 3]
```diff
@@ -1,3 +0,0 @@
-- Remove BlockID from Commit
-- Actually validate the ValidatorsHash
-- Move blockHeight=1 exception for LastCommit to ValidateBasic
```
[the remark this answers is not in the document]
Comment (zramsay) [reply docs/specification/new-spec/spec-notes.md]: #1152

**Comment by ebuchman** [comment]
rebased and merged on develop

**Pull request closed without merging**